src/store/actions/causes.js

```diff
@@ -1,9 +1,42 @@
 import { ACTION_TYPES } from '../actionTypes';
 
+
+const dataSanitization = data => {
+  if (data) {
+    const {
+      id,
+      causes,
+      causes_heading: { heading_primary, heading_secondary }
+    } = data;
+    const sanitizedCauses = causes.map(
+      ({ title, description, raised, goal, image: { url: imgURL }, id }) => {
+        return {
+          title,
+          description,
+          raised,
+          goal,
+          imgURL,
+          id
+        };
+      }
+    );
+    return {
+      id,
+      causes: sanitizedCauses,
+      causes_heading: {
+        heading_primary,
```

Comment:
I'm noticing that some fields are using snake_case naming convention, while others are using camelCase, can we agree on a single convention?

Reply:
We do not explicitly follow a style guide in naming conventios. 
I will consider it. Thanks 👍